frame/staking/src/tests.rs

```diff
@@ -828,7 +827,7 @@ fn reward_destination_works() {
 		<Module<Test>>::reward_by_ids(vec![(11, 1)]);
 
 		start_era(1);
-		mock::make_all_reward_payment(0);
```

Comment:
actually I prefer having these functions explicitly imported from mock so it shows this is just util function but I'm ok.

Reply:
I added it back, you make a good point.